xbps-src

```diff
@@ -9,7 +9,9 @@ print_cross_targets() {
 }
 
 usage() {
-    cat << _EOF
+while read ; do
+    printf "%s\n" "$REPLY"
+done <<- _EOF
```

Comment:
`cat` is fine here, beside a short and rare usage of `usage`, `cat` is easier to reason about.

Reply:
why using something outside `bash` if it have the solution